cli.js

```diff
@@ -73,8 +73,20 @@ function filterProcesses(input, processes, flags) {
 		});
 }
 
+function handleFkillError(proc) {
+	inquirer.prompt([{
+		type: 'confirm',
+		name: 'forceKill',
+		message: 'Error killing process. Would you like to use the force?'
+	}]).then(answer => {
+		if (answer.forceKill === true) {
+			return fkill(proc, {force: true});
+		}
+	});
+}
+
 if (cli.input.length === 0) {
 	init(cli.flags);
 } else {
-	fkill(cli.input, cli.flags);
+	fkill(cli.input, cli.flags).catch(() => handleFkillError(cli.input));
```

Comment:
The prompt should only show when the terminal is interactive (`process.stdout.isTTY`), otherwise it should just print what processes it couldn't kill and recommend the `--force` flag.